xml/System.IO/StreamWriter.xml

```diff
@@ -1552,10 +1552,18 @@ For a list of common I/O tasks, see [Common I/O Tasks](~/docs/standard/io/common
         <Parameter Name="arg0" Type="System.Object" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="format">To be added.</param>
-        <param name="arg0">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="format">A composite format string.</param>
+        <param name="arg0">The object to format and write.</param>
+        <summary>Writes a formatted string to the stream, using the same semantics as the <see cref="M:System.String.Format(System.String,System.Object)" /> method.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+See <xref:System.IO.TextWriter.Write(System.String,System.Object)> for a description of the composite formatting capabilities offered.
```

Comment:
@rpetrusha is it ok to reference its parent's implementation instead of copy-pasting the whole remark?

Reply:
Yes, it is, @carlossanlop.